kaznet/apps/ona/api.py

```diff
@@ -359,8 +359,7 @@ def sync_updated_instances(form_id: int):
     else:
         raw_ids = fetch_form_data(
             formid=the_xform.ona_pk,
-            dataids_only=True,
-            edited_only=True)
+            dataids_only=True)
```

Comment:
I removed this parameter because when this function is called with it an empty array is returned and therefore no instance that should be updated is ever updated. with this, ids for all  instances for this form will be returned and then processed one by one. I recognize that may be a performance problem when there are serveral instances for a given form. and maybe a separate issue should be created to fix this.

Reply:
Have you created that separate issue that you mention here?

Reply:
yes: #351